R/expr-meta.R

```diff
@@ -34,14 +62,22 @@ expr_meta_output_name <- function(..., raise_if_undetermined = TRUE) {
       tryCatch(
         self$`_rexpr`$meta_output_name(),
         error = function(e) {
-          NULL
+          NA_character_
```

Comment:
Why this change?

Reply:
Because it returns `None` in python, which we usually replace by `NA` throughout the docs so I think it makes more sense (it's also what we have in the current r-polars). I can change the call to is.null() if you're ok with that.

Reply:
I see. Please update the following to use `is_na(expr_name)`.
https://github.com/eitsupi/neo-r-polars/blob/d72e64b92871a2a9b0879f0ec35025dae6c314d5/R/utils-parse-expr.R#L11-L14